### PR TITLE
New dasgoclient version

### PR DESCRIPTION
**Pull request by vkuznet**

### CODE REVIEW

**Comment by smuzaffar** [comment]
please

**Comment by mrodozov** [comment]
please test

File: dasgoclient.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### RPM cms dasgoclient v02.00.06
+### RPM cms dasgoclient v02.00.07
 ## NOCOMPILER
 %define dasgoclient_arch     slc6_amd64_gcc630
```
Comment (smuzaffar) [inline dasgoclient.spec:3]: @vkuznet , default compiler is gcc700 now, we should update the architecture here to slc6_amd64_gcc700 and build  dasgoclient-binary for slc6_amd64_gcc700.

**Comment by vkuznet** [comment]
Shahzad,
I made necessary changes and created a new PR: https://github.com/cms-sw/cmsdist/pull/4295
I'm closing this one in favor of new one.
Best,
Valentin.

**Pull request closed without merging**